test/testcredentialmanager.cpp

```diff
@@ -81,7 +81,7 @@ private Q_SLOTS:
                 });
             });
         });
-        QTest::qWaitFor([this] { return _finished; });
+        QVERIFY(QTest::qWaitFor([this] { return _finished; }));
```

Comment:
@TheOneRing could you please comment on that? I expect the test execution has to wait until job will be finished, right? Or I missed something...

Reply:
Yes the idea is to wait for the test to finish.
As we have the skip condition on linux the wait might time out here.